CONTRIBUTING.md

```diff
@@ -193,6 +193,20 @@ We use automated whitespace convention checking. Violations can be fixed by
 running [fix-whitespace](https://hackage.haskell.org/package/fix-whitespace). If
 you push a fix of a whitespace violation, please do so in a _separate commit_.
 
+API Changes
+-----------
+
+The `Check API` job tests the `Cabal` and `Cabal-syntax` packages for API changes.
+It's still something of a work in progress, but even in its current state it is
```

Comment:
I feel like "work in progress" may suggest someone that it can be ignored. Since I don't see what this phrase achieves, I'd not put it here. Generally, if the idea won't work as smooth as we hope, we can always nuke it even if it's not a WIP...

Reply:
My intent was to suggest that this section is subject to change and contributors should check back to see if it has — but I expect such changes if any to happen relatively quickly.